opm/common/OpmLog/TimerLog.hpp

```diff
@@ -50,7 +50,6 @@ class TimerLog : public StreamLog {
                                    const std::string& message) override;
 private:
     clock_t m_start;
-    std::ostringstream m_work;
```

Comment:
Cool.  Do you plan to move [the `<sstream>` header](https://github.com/OPM/opm-common/blob/91e2a1256e5cf0ef0dc51e4c9be0ccf37a12e91d/opm/common/OpmLog/TimerLog.hpp#L25) to the implementation file too?

Reply:
Yeah, that was sort of the whole point. Bad git add -p work on my behalf..